src/main/java/domain/generatestrategy/ManualLotteryGeneratorStrategy.java

```diff
@@ -0,0 +1,28 @@
+package domain.generatestrategy;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import domain.Lottery;
+import domain.LotteryNumber;
+import domain.factory.LotteryNumberFactory;
+
+public class ManualLotteryGeneratorStrategy implements LotteryGenerateStrategy {
+
+	private final LotteryNumberFactory lotteryNumberFactory;
+	private final List<Integer> rawManualLotteries;
+
+	public ManualLotteryGeneratorStrategy(final List<Integer> rawManualLotteries, final LotteryNumberFactory lotteryNumberFactory) {
+		this.lotteryNumberFactory = lotteryNumberFactory;
+		this.rawManualLotteries = rawManualLotteries;
+	}
+
+	@Override
+	public Lottery getLottery() {
+		List<LotteryNumber> manualLotteries = rawManualLotteries.stream()
+			.map(lotteryNumberFactory::of)
+			.collect(Collectors.toList());
+		return new Lottery(manualLotteries);
+	}
```

Comment:
List<Integer> rawManualLotteries로 한개의 Lottery 번호만 가지고 있는데 변수명이 애매하지 않을까요??
그리고 질문에서 매번 ManualLotteryGeneratorStrategy를 생성한다고 했는데, 처음에 List<List<Integer>>를 가지고 있고,
메서드가 호출될 때마다 반환할 수 있지 않을까요?
```suggestion
         private final List<List<Integer>> rawManualLotteries;
	@Override
	public Lottery getLottery() {
	        List<Integer> rawManualLotteryNumbers = rawManualLotteries.remove(0);
		List<LotteryNumber> manualLotteryNumbers = rawManualLotteryNumbers.stream()
			.map(lotteryNumberFactory::of)
			.collect(Collectors.toList());
		return new Lottery(manualLotteryNumbers);
	}
```

Reply:
저걸 생각을 못했네요... 수정하겠습니다.